packages/calypso-analytics/src/utils/current-user.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Internal dependencies
+ */
+import hashPii from './hash-pii';
+
+/**
+ * Module variables
+ */
+let _currentUser: CurrentUser;
+
+export interface CurrentUserHashedPii {
+	ID: string;
+	username: string;
+	email: string;
+}
+
+export interface CurrentUser {
+	ID: number;
+	username: string;
+	email: string;
+	hashedPii?: CurrentUserHashedPii;
+}
+
+/**
+ * Gets current user.
+ *
+ * @returns {object|undefined} Current user.
+ */
+export function getCurrentUser(): CurrentUser {
+	return _currentUser;
+}
+
+/**
+ * Sets current user, (stored in javascript memory).
+ *
+ * @param {object} currentUser the user data for the current user
+ * @returns {object|undefined} Current user.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function setCurrentUser( currentUser: any ): CurrentUser | undefined {
```

Comment:
I was wondering if it'd be better to just define a wee interface for this function's parameter type. Something like
```typescript
interface SetCurrentUserParam {
	ID: string;
	username: string;
	email: string;
}
```

It doesn't need to be exported.